tests/vision/checks/distribution/image_property_drift_test.py

```diff
@@ -37,7 +37,22 @@ def test_image_property_drift_check(coco_train_visiondata, coco_test_visiondata,
 
 def test_image_property_drift_check_limit_classes(coco_train_visiondata, coco_test_visiondata, device):
     # Run
-    result = ImagePropertyDrift(classes_to_display=['bicycle', 'bench', 'bus', 'truck'], min_samples=5
+    result = ImagePropertyDrift(categorical_drift_method='PSI',
+                                classes_to_display=['bicycle', 'bench', 'bus', 'truck'], min_samples=5
+                                ).run(coco_train_visiondata, coco_test_visiondata, device=device)
+
+    # Assert
+    assert_that(result, is_correct_image_property_drift_result())
+
+    assert_that(result.value, has_entries(
+        {'Brightness': close_to(0.13, 0.01)}
+    ))
```

Comment:
Currently, all image properties are continuous / numeric. The categorical_drift_method parameter should have no effect